core/config/default-config.js

```diff
@@ -541,6 +542,7 @@ const defaultConfig = {
         {id: 'geolocation-on-start', weight: 1, group: 'best-practices-trust-safety'},
         {id: 'notification-on-start', weight: 1, group: 'best-practices-trust-safety'},
         {id: 'csp-xss', weight: 0, group: 'best-practices-trust-safety'},
+        {id: 'has-hsts', weight: 0},
```

Comment:
Forgot we changed this haha, sorry for the misinformation 
```suggestion
        {id: 'has-hsts', weight: 0, group: 'hidden'},
```

Reply:
Thanks, done!